doc/aurbuild.md

```diff
@@ -0,0 +1,31 @@
+# aurbuild
+
+## Name
+
+aurbuild - Build packages to a local repository
+
+## Synopsis
+
+aurbuild [+-a ARG] [+-cd ARG] [+-p ARG] [+-r ARG} [--] ARGS...
+
+## Description
+
+Build a package from a queue file and add them to a local repository. A local repository is a repository on the local file system. It is configured using the file:// prefix for Server in pacman.conf.
+
+## Operations
+
+* __-a__ A text file with directories containing PKGBUILDs (
+* __-c__ Build in a chroot
+* __-d__ The name of the database
+* __-p__ Select the pool for the built packages. The pool is the location where aurbuild will put the packages after building
+* __-r__ Select the root for the repository where the database files will live
+
+All arguments after -- get passed to makepkg
```

Comment:
Nitpick: s/get/are. Also, 3e3ba2410389581398bff5fb0d7c60f64c5a9c3c allowed to pass arguments to makechrootpkg, when `-c` is provided.